filesec/file_security.go

```diff
@@ -423,8 +423,16 @@ func (s *FileSecurity) VerifyCertificate(certpem []byte, name string) error {
 		return err
 	}
 
+	intermediates := x509.NewCertPool()
+	if !intermediates.AppendCertsFromPEM(certpem) {
+		s.log.Warnf("Could not add intermediates: %s", err)
```

Comment:
no err being set by these methods so we cant log its contents - would be nil - i guess we should just continue if its false since it gives us no details and what was there might not have intermediates.